lib/kubernetes-deploy/logger.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+require 'logger'
```

Comment:
I guess this is ok for now. I'd still like to hear Kir's thoughts when he's available, but we can implement them in a follow-up.